src/styles/theme.js

```diff
@@ -0,0 +1,61 @@
+import searchIcon from '../../public/loupe.png';
+import { COLORS, THEMES } from '../common/constants';
+
+const light = {
+  primaryColor: COLORS.white_regular,
+  primaryColorDarker: COLORS.white_dark,
+  complimentaryColor: COLORS.blue_regular,
+  secondaryColor: COLORS.grey_light,
+  secondaryColorDarker: COLORS.grey_regular,
+  errorColor: COLORS.red_regular,
+  fontColor: COLORS.grey_dark
+};
+
+const dark = {
+  primaryColor: COLORS.grey_dark,
+  primaryColorDarker: COLORS.black_regular,
+  complimentaryColor: COLORS.blue_regular,
+  secondaryColor: COLORS.white_regular,
+  secondaryColorDarker: COLORS.white_dark,
+  errorColor: COLORS.red_regular,
+  fontColor: COLORS.white_dark
+};
+
+export const themesMap = new Map([
+  [THEMES.dark, dark],
+  [THEMES.light, light]
+]);
+
+export const getTheme = mode => ({
+  colors: mode,
+  fontSize: {
```

Comment:
not sure about keeping all the static values within theme object. You are able to change your small width value in one place and it will be changed everywhere.... it is for you to decide whether it is super good or super bad. Do not overconfigure it.

Reply:
Keeping all values in variables was an idea we discussed with Igor, when he was our trainer, that I really liked.  The biggest benefit for me is the fact that I don't need to remember/look up values in order to create consistent styles

Reply:
ok, if it makes your work easier - that's good. But I really doubt that you never look up there when choosing between, say,  s and xs.